apps/complaints/acceptance/pages/confirmation.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = {
-  url: 'confirmation'
+  url: 'confirmation',
+  content: {
+    sent: 'The complaint has been sent.',
```

Comment:
@JoeChapman not sure how I feel about this, as this is just for acceptance tests it would add complexity. I am more for not testing content at all on the page, as this can change